internal/configurations/configurations.go

```diff
@@ -212,6 +215,41 @@ func (s *Configuration) GetSecret(ctx context.Context) (*v1.Secret, error) {
 	return secret, nil
 }
 
+// LabelServiceSecrets will look for the Opaque secrets released with a service, looking for the
+// app.kubernetes.io/instance label, then it will add the Configuration labels to "create" the configurations
+func LabelServiceSecrets(ctx context.Context, kubeClient *kubernetes.Cluster, namespace, name string) ([]v1.Secret, error) {
+	secretSelector := labels.Set(map[string]string{
+		"app.kubernetes.io/managed-by": "Helm",
+		"app.kubernetes.io/instance":   name,
```

Comment:
I think this needs to be `models.ServiceHelmChartName(name, namespace)` because we had to prefix the name of the helmchart with the target namespace to avoid collisions when 2 services with the same name exist on 2 different namespaces (the helmcharts are all created in the `epinio` namespaces because of how the helm-controller works). CI being green means we don't test this, it should be broken.